src/garage/np/algos/meta_rl_algorithm.py

```diff
@@ -0,0 +1,35 @@
+"""Interface of Meta-RL ALgorithms."""
+import abc
+
+from garage.np.algos import RLAlgorithm
+
+
+class MetaRLAlgorithm(RLAlgorithm, abc.ABC):
+    """Base class for Meta-RL Algorithms."""
+
+    @abc.abstractmethod
+    def get_exploration_policy(self):
+        """Return a policy used before adaptation to a specific task.
+
+        Each time it is retrieved, this policy should only be evaluated in one
+        task.
+
+        Returns:
+            garage.Policy: The "un-adapted" policy.
```

Comment:
Hmm i think this could be more specific. You can say that it's the policy used to obtain samples which are later used for meta-RL adaptation. It's up to the algorithm to decide what this policy will be (for instance, it could be a random walk).